app/views/posts/show.html.erb

```diff
@@ -1,10 +1 @@
-<p style="color: green"><%= notice %></p>
-
-<%= render @post %>
-
-<div>
-  <%= link_to "Edit this post", edit_post_path(@post) %> |
-  <%= link_to "Back to posts", posts_path %>
-
-  <%= button_to "Destroy this post", @post, method: :delete %>
-</div>
+<h1>Here is a list of posts for a given user</h1>
```

Comment:
**OPTIONAL** - The show page shows the details about a post, while the index page shows all posts for a particular user, I suggest you fix the placeholders 👍 👍